tests/python/tests/migration_php8/test_comparison.py

```diff
@@ -0,0 +1,81 @@
+import os
+from typing import List
+
+from python.lib.testcase import KphpCompilerAutoTestCase
+
+from python.lib.colors import green, red, yellow
+
+
+class TestPHP8NumberStringComparison(KphpCompilerAutoTestCase):
```

Comment:
Take a look at _**kphp_should_fail**_ and _**kphp_should_warn**_ tags. I strongly recommend add the same for runtime warnings and use it instead of the function python tests.

Reply:
Done, added `@kphp_runtime_should_warn` tag.